product_docs/docs/pgd/5/sequences.mdx

```diff
@@ -18,8 +18,8 @@ produce values that are unique only on the local node. This is important because
 unique ids generated by such sequences cause conflict and data loss by
 means of discarded `INSERT` actions in multi-master replication.
 
-!!! Note Permissions Required
-PGD global sequences are on by default. This means that for any user to use sequences, 
+!!! Note Permissions required
+PGD global sequences are on by default. This means that, to use sequences, 
```

Comment:
This note is making the point that for ANY user to use sequences they must have the appropriate role. Sequences are embedded and other users using the table are required to have the same role privilege - hence for any user rather than the user.